numba/tests/test_nested_calls.py

```diff
@@ -148,6 +154,14 @@ def test_call_generated(self):
         self.assertPreciseEqual(cfunc(1, 2), (-4, 2))
         self.assertPreciseEqual(cfunc(1j, 2), (1j + 5, 2))
 
+    def test_nested_annotated(self):
+        """
+        Tested a nested function with annotations.
+        """
+        cfunc = njit()(nested_annotated)
+        # should not fail
+        cfunc()
```

Comment:
This seems like a bit of a strange place for a test relating to the front end byte code processing. It also doesn't assert anything. However, given the context it's doing what is needed.